Doc/whatsnew/3.14.rst

```diff
@@ -110,6 +110,21 @@ ast
 
   (Contributed by Bénédikt Tran in :gh:`121141`.)
 
+dis
+---
+
+* Added support for rendering :class:`instruction positions <dis.Positions>`
+  when available. This feature is added to the following interfaces via
```

Comment:
```suggestion
* Added support for rendering full source location information of instructions,
 rather than only the line number. This feature is added to the following interfaces via
```